test.properties

```diff
@@ -648,8 +648,7 @@
         gradle-plugins-gulp,\
         gradle-plugins-js-module-config-generator,\
         gradle-plugins-js-transpiler,\
-        gradle-plugins-node,\
-        gradle-plugins-workspace
+        gradle-plugins-node
```

Comment:
So this means that the "gradle-plugins" will run everything but the ones here that are excluded?

Reply:
yes. these were excluded from before. Andrea was getting daily emails about the ones failing.